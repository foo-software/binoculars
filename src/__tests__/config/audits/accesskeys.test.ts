import auditAccesskeys from '../../../config/audits/accesskeys';

describe('accesskeys audit', () => {
  it('should match snapshot', () => {
    expect(auditAccesskeys('en')).toMatchSnapshot();
  });
});
