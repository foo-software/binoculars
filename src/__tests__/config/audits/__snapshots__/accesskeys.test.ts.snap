// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`accesskeys audit should match snapshot 1`] = `
Object {
  "description": "Access keys let users quickly focus a part of the page. For proper navigation, each access key must be unique. [Learn more](https://web.dev/accesskeys/).",
  "title": "\`[accesskey]\` values are unique",
}
`;
